src/DynamoCoreWpf/ViewModels/Core/NoteViewModel.cs

```diff
@@ -140,7 +141,20 @@ private void CreateGroup(object parameters)
 
         private bool CanCreateGroup(object parameters)
         {
-            return DynamoSelection.Instance.Selection.OfType<ModelBase>().Any();
+            var groups = WorkspaceViewModel.Model.Annotations;
+            //Create Group should be disabled when a group is selected
+            if (groups != null && groups.Any(x => x.IsSelected))
+            {
```

Comment:
It looks like you didn't update all of the code to remove this null check.  It should be a guarantee that this property is never null.